stress/v2/statement/go.go

```diff
@@ -0,0 +1,35 @@
+package statement
+
+import (
+	"fmt"
+
+	"github.com/influxdata/influxdb/stress/v2/ponyExpress"
+)
+
+// GoStatement is a Statement Implementation to allow other statements to be run concurrently
+type GoStatement struct {
+	Statement
+
+	StatementID string
+}
+
+// SetID statisfies the Statement Interface
+func (i *GoStatement) SetID(s string) {
+	i.StatementID = s
+}
+
+// Run statisfies the Statement Interface
+func (i *GoStatement) Run(s *ponyExpress.StoreFront) {
+	s.Add(1)
+	s.Lock()
+	go func() {
+		s.Unlock()
```

Comment:
Sinc `s.Unlock()` is at the top of the anonymous function you don't seem to be protecting access to anything.